docs/pages/enterprise/sso.mdx

```diff
@@ -32,9 +36,149 @@ organization's Single Sign-On (SSO) provider.
   </Tile>
 </TileSet>
 
-## How does SSO work?
+## How Teleport uses SSO
+
+You can register your Teleport cluster as an application with your SSO provider.
+When a user signs in to Teleport, your SSO provider will execute its own
+authentication flow, then send an HTTP request to your Teleport cluster to
+indicate that authentication has completed.
+
+Teleport authenticates users to your infrastructure by issuing short-lived
+certificates. After a user completes an SSO authentication flow, Teleport issues
+a short-lived certificate to the user. Teleport also creates a temporary `user`
+resource on the Auth Service backend.
+
+### Temporary `user` resources
+
+After a user completes an SSO authentication flow, Teleport creates a temporary
+`user` resource for the user.
+
+When a user signs in to Teleport with `tsh login`, they can configure the TTL of
+the `user` Teleport creates. Teleport enforces a limit of 30 hours (the default
+is 12 hours).
+  
+In the Teleport audit log, you will see an event of type `user.create` with
+information about the temporary user.
+
+<Details title="How can I inspect a temporary user resource?">
+
+You can inspect the temporary `user` resource created via your SSO integration
+by using the `tctl` command:
+
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+
+```code
+# Log in to your cluster with tsh so you can use tctl remotely
+$ tsh login --proxy=proxy.example.com --user=myuser
+$ tctl get users
+```
 
-Execute the following command to log in to your Teleport cluster using the CLI.
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+
+```code
+# Log in to your cluster with tsh so you can use tctl remotely
+$ tsh login --proxy=mytenant.teleport.sh --user=myuser
+$ tctl get users
+```
+</ScopedBlock>
+
+Here is an example of a temporary `user` resource created when the GitHub user
+`myuser` signed in to GitHub to authenticate to Teleport. This resource
+expires 12 hours after creation. The `created_by` field indicates that the
+resource was created by Teleport's GitHub SSO integration:
+
+```yaml
+kind: user
+metadata:
+  expires: "2022-06-15T04:02:34.586688054Z"
+  id: 0000000000000000000
+  name: myuser
+spec:
+  created_by:
+    connector:
+      id: github
+      identity: myuser
+      type: github
+    time: "2022-06-14T16:02:34.586688441Z"
+    user:
+      name: system
+  expires: "0001-01-01T00:00:00Z"
+  github_identities:
+  - connector_id: github
+    username: myuser
+  roles:
+  - editor
+  - access
+  - auditor
+  status:
+    is_locked: false
+    lock_expires: "0001-01-01T00:00:00Z"
+    locked_time: "0001-01-01T00:00:00Z"
+    recovery_attempt_lock_expires: "0001-01-01T00:00:00Z"
+  traits:
+    github_teams:
+    - my-team
+    kubernetes_groups: null
+    kubernetes_users: null
+    logins:
+    - root
+version: v2
+```
+
+</Details>
+
+### Certificates for SSO users
+
+Along with creating a temporary `user` resource, Teleport issues an X.509
```

Comment:
Technically it generates an SSH certificate for the user also, not just X.509.

Reply:
Added information about the SSH certificate